server/views/components/page-description/index.njk

```diff
@@ -0,0 +1,10 @@
+<header class="page-description {% for modifier in modifiers %}page-description--{{ modifier }} {% endfor %}">
+  <span class="page-description__intro {% for modifier in modifiers %}page-description__intro--{{ modifier }} {% endfor %}">{{ intro }}</span>
+  <h1 class="page-description__title {% for modifier in modifiers %}page-description__title--{{ modifier }} {% endfor %}">{{ title }}</h1>
+  <span class="page-description__outro {% for modifier in modifiers %}page-description__outro--{{ modifier }} {% endfor %}">
```

Comment:
Yeah, this is rubbish isn't it? I think in an attempt to cover off all your points, I'm increasingly unhappy with what's going on and I'm wasting your time a bit.

I'm more in favour of going back to the component-scoped style that I started out with. 

You mentioned above that we had decided not to go down that route, but I wasn't aware of that – I was very much under the impression that we were going to take a component driven approach.

Would you object to me having another bash at it that way round?